packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -351,6 +355,13 @@ class ExpansionPanelList extends StatefulWidget {
   /// constructor.)
   final Object initialOpenPanelValue;
 
+  /// Padding is applied to panel header when expanded.
```

Comment:
The format we use for API documentation is as follows:
```dart
/// One sentence providing an overview of the property/method
///
/// More description on the property/method. This can usually span multiple
/// lines. 
///
/// New paragraphs are typically separated by a newline between the paragraphs.
```